### PR TITLE
Add resources alias

### PR DESCRIPTION
**Pull request by soulshake**
* Add alias `resources` to `services` command.
* Update usage strings to prefer `resources`.
* Add `resources_test.go`.

### CODE REVIEW

**Comment by soulshake** [comment]
[Deprecate `convox services`, replace with `convox resources`](https://trello.com/c/Kg8TJiCY/1102-deprecate-convox-services-replace-with-convox-resources)

**Comment by soulshake** [comment]
Regarding this failing test, see comment thread on #1399.

**Comment by soulshake** [comment]
[Guide: Deploy / Resources](https://trello.com/c/u5hs7ksl/1168-guide-deploy-resources)

**Comment by nzoschke** [comment]
- [x] `convox resources` works
- [ ] `convox resources help` shouldn't say "service"

File: cmd/convox/services.go [hunk 1]
```diff
@@ -67,7 +67,8 @@ func init() {
 
 	stdcli.RegisterCommand(cli.Command{
 		Name:        "services",
-		Description: "manage services",
+		Aliases: []string{"resources"},
```
Comment (nzoschke) [inline cmd/convox/services.go]: Nice, didn't know you could do this! Can you implement it the other way, i.e. resources aliases "services"? That'd make `convox help` bit a bit cleaner

```
$ convox help | grep resources
  services, resources	manage resources [prev. services]
```

File: cmd/convox/services.go [hunk 2]
```diff
@@ -66,66 +67,67 @@ func init() {
...
+		Aliases: []string{"services"},
+		Description: "manage external resources [prev. services]",
 		Usage:       "",
 		Action:      cmdServices,
 		Flags:       []cli.Flag{rackFlag},
 		Subcommands: []cli.Command{
 			{
 				Name:            "create",
-				Description:     "create a new service.",
+				Description:     "create a new resource.",
```
Comment (nzoschke) [inline cmd/convox/services.go]: not your fault, but I noticed some descriptions have a period and some don't. Can we converge on one pattern?

Comment (soulshake) [inline cmd/convox/services.go]: Created a [Trello card](https://trello.com/c/4MutugIf/1204-standardize-use-of-periods-in-cli-usage-strings).

**Comment by soulshake** [comment]
@nzoschke Thanks, added those tasks to [the Trello card](https://trello.com/c/Kg8TJiCY/1102-deprecate-convox-services-replace-with-convox-resources).

File: cmd/convox/resources_test.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            Command: "convox resources create syslog --name=syslog-1234 --url=tcp+tls://logs1.example.com:12345",
+            Exit:    0,
+            Stdout:  "Creating syslog-1234 (syslog: name=\"syslog-1234\" url=\"tcp+tls://logs1.example.com:12345\")... CREATING\n",
+        },
+    )
+}
+
+func TestResourcesDelete(t *testing.T) {
+
+	ts := testServer(t,
```
Comment (MiguelMoll) [inline cmd/convox/resources_test.go]: This extra test server can be removed. `tsd` has the Delete method that works 👍

File: cmd/convox/services_test.go [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            Command: "convox services create syslog --name=syslog-1234 --url=tcp+tls://logs1.example.com:12345",
+            Exit:    0,
+            Stdout:  "Creating syslog-1234 (syslog: name=\"syslog-1234\" url=\"tcp+tls://logs1.example.com:12345\")... CREATING\n",
+        },
+    )
+}
+
+func TestServicesDelete(t *testing.T) {
+
+	ts := testServer(t,
```
Comment (MiguelMoll) [inline cmd/convox/services_test.go]: This extra test server can be removed. `tsd` has the Delete method that works 👍

**Comment by MiguelMoll** [comment]
Once the extra test servers are removed, this lgtm 👍

**Comment by mattmanning** [comment]
[Add resources alias to CLI](https://trello.com/c/6218Kt21/1259-add-resources-alias-to-cli)

**Pull request closed without merging**